.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v2.1.3
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+      - run: npm install --prefix ui
+      - run: npm run lint --prefix ui
+      - run: npm run build --prefix ui
+      - run: mv ui/build/ui-bundle.zip ui/build/ui-bundle-${UI_BUNDLE_VERSION}.zip
+        env:
+          UI_BUNDLE_VERSION: ${{ github.ref }}
```

Comment:
The release process is:

- install dependencies
- lint (just in case)
- build
- rename the `build/ui-bundle.zip` file to add the version

If all goes well, when we push tag `v1.0.0` it should upload to S3 an archive named: `ui-bundle-v1.0.0.zip`